fitbenchmarking/fitbenchmarking
# 333 results object

#### Description of Work

Fixes #333 

Processes the data within the results object to be viewed in the table. 

#### Testing Instructions

1. Check output still the same

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
